src/backend/core/tests/documents/test_api_document_invitations.py

```diff
@@ -417,94 +417,11 @@ def test_api_document_invitations_create_privileged_members(
         }
 
 
-def test_api_document_invitations_create_email_from_content_language():
```

Comment:
I think we should put back this test with the updated behavior (from sender language).

Reply:
✅